library/returnBook/ReturnBookUI.java

```diff
@@ -4,33 +4,51 @@
 
 public class ReturnBookUI {
 
-	public static enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+	//public static enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+	//public static enum ui_state { INITALISD, READY, INSPECTING, COMPLETED};
+	public static enum UiState { INITALISD, READY, INSPECTING, COMPLETED};    //ui_state changed to UiState  : Reviewer : Ankush
 
-	private rETURN_bOOK_cONTROL CoNtRoL;
-	private Scanner iNpUt;
-	private uI_sTaTe StATe;
+	//private rETURN_bOOK_cONTROL CoNtRoL;
+	//private return_book control;
+	private ReturnBookControl control;    //return_book control changed to ReturnBookControl control:   by :REV : Ankush
+	//private Scanner iNpUt;
+	private Scanner input;
+	//private uI_sTaTe StATe;
+	private UiState state;
 
 	
-	public ReturnBookUI(rETURN_bOOK_cONTROL cOnTrOL) {
-		this.CoNtRoL = cOnTrOL;
-		iNpUt = new Scanner(System.in);
-		StATe = uI_sTaTe.INITIALISED;
-		cOnTrOL.sEt_uI(this);
+	//public ReturnBookUI(rETURN_bOOK_cONTROL cOnTrOL) {
+	//public ReturnBookui(return_book_control control) {
+	public ReturnBookUi(ReturnBookControl control)     //return_book_control control) changed to ReturnBookControl control  by REV: Ankush
+		//this.CoNtRoL = cOnTrOL;
+		this.control = control;
+		//iNpUt = new Scanner(System.in);
+		input = new Scanner(System.in);
+		//StATe = uI_sTaTe.INITIALISED;
+		//State = ui-state.INITIALISED;
+		state = UiState.INITIALISED;  //State = ui-state.INITIALISED changed to state = UiState.INITIALISED by Rev: Ankush
+		//cOnTrOL.sEt_uI(this);
+		//control.set_ui(this);
+		control.setUi(this);   //control.set_ui(this) changed to control.setUi(this) by Rev: Ankush
 	}
 
 
-	public void RuN() {		
-		oUtPuT("Return Book Use Case UI\n");
+	//public void RuN() {
+        public void Run() {
+		//oUtPuT("Return Book Use Case UI\n");
+		output("Returnbook use case ui\n")
 		
 		while (true) {
 			
-			switch (StATe) {
+			//switch (StATe) {
+			switch (state) {
 			
 			case INITIALISED:
 				break;
 				
 			case READY:
-				String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
+				//String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
+				String BookInputString = input("Scan book (<enter> completes): ");

```

Comment:
a variable name should be bookInputString instead of BookInputString 
@dipanshu42